vagrant/destroy.sh

```diff
@@ -22,3 +22,4 @@ fi
 
 # 3. Destroy bootstrap concourse
 vagrant destroy -f
+rm .kernel-updated
```

Comment:
This won't cover the case when the machine is deleted by the `self-terminate` pipeline.